test/kwokctl/suite.sh

```diff
@@ -21,25 +21,7 @@ function show_all() {
 
 function show_info() {
   local name="${1}"
-  echo
-  echo kwokctl --name="${name}" kubectl get pod -o wide --all-namespaces
-  kwokctl --name="${name}" kubectl get pod -o wide --all-namespaces
-  echo
-  echo kwokctl --name="${name}" logs etcd
-  kwokctl --name="${name}" logs etcd
-  echo
-  echo kwokctl --name="${name}" logs kube-apiserver
-  kwokctl --name="${name}" logs kube-apiserver
-  echo
-  echo kwokctl --name="${name}" logs kube-controller-manager
-  kwokctl --name="${name}" logs kube-controller-manager
-  echo
-  echo kwokctl --name="${name}" logs kube-scheduler
-  kwokctl --name="${name}" logs kube-scheduler
-  echo
-  echo kwokctl --name="${name}" logs kwok-controller
-  kwokctl --name="${name}" logs kwok-controller
-  echo
+  kwokctl --name="${name}" export logs /tmp/kwok/logs
```

Comment:
```suggestion
  kwokctl --name="${name}" export logs "/tmp/kwok/logs/${name}"
```

Reply:
I make cluster name to be the base dir as follows:

```
└── kwok-export-logs-cluster-docker-1-27-2
    ├── components
    │   ├── audit.log
    │   ├── etcd.log
    │   ├── kube-apiserver.log
    │   ├── kube-controller-manager.log
    │   ├── kube-scheduler.log
    │   └── kwok-controller.log
    ├── docker-info.txt
    └── kwok.yaml
```